defrag/routes/obs_packages.py

```diff
@@ -0,0 +1,16 @@
+from defrag.modules.helpers import Query, QueryResponse
+from defrag.modules.helpers.cache_manager import Memo_Redis
+from defrag.modules.obs_packages import search
+
+from fastapi import APIRouter
+router = APIRouter()
+
+__ENDPOINT_NAME__ = "obs_packages"
+
+
+@router.get("/" + __ENDPOINT_NAME__ + "/search/{distribution}")
+@Memo_Redis.install_decorator("/" + __ENDPOINT_NAME__ + "/search/")
+async def api_search(keywords: str, distribution: str, home_repos: bool = False) -> QueryResponse:
```

Comment:
This does not work:
```
$ curl -X 'GET' \
  'http://localhost:8000/obs_packages/search/tumbleweed?keywords=spotify-qt&home_repos=true' \
  -H 'accept: application/json'
{"query":{"service":"obs_packages","item_id":null},"results_count":0,"results":[],"error":null,"message":null}

Reply:
This is "fixed" by removing some filters, done in the last commit.